### DISCUSSION TITLE
setup expense project in ansible

### DISCUSSION THREAD

**Discussion by gopi-chowdhary** [learndevopsonline/learndevopsonline#1502] [Q&A]
workstation ip : 54.90.68.163
Error: failed to connect to the host
![Screenshot (9)](https://github.com/learndevopsonline/learndevopsonline/assets/145432552/ae169137-ea6a-41ee-ac21-3796ee8d1129)

**Comment by sreedharm07** [comment]
@gopi-chowdhary 
your workstation is inactive to check

**Reply by gopi-chowdhary** [reply]
workstation is in running state 
![Screenshot (10)](https://github.com/learndevopsonline/learndevopsonline/assets/145432552/eb7c598d-5ee1-46a2-be17-e8152799103d)

**Reply by sreedharm07** [reply]
Hi @gopi-chowdhary 
its working fine with your frontend ip  (ansible-playbook -i **3.80.155.249**, -e ansible_user=centos -e ansible_password=DevOps321 expense.yml -e role_name=frontend) , but not with your dns record. 
 we can assume that error is because of your dns. your dns was wrong or its unavailable  or not created dns name with proper ip. please check your dns records.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/f5316d6c-373c-446c-ab81-1ef4dcc2f7f4)
